### ISSUE TITLE
Replicate the Google Books API for testing

### ISSUE THREAD

**Issue by KPull** [KPull/Bastion#26]
One of our tests, `GoogleBooksApiTest`, uses the actual Google API endpoint for testing. Unfortunately, the has been found to change. We need to replicate the responses using our own embedded server and point the `GoogleBooksApiTest` to our embedded test server.

**Comment by KPull** [comment]
I've marked `GoogleBooksApiTest` as ignored for the meantime. Remove the `@Ignored` annotation when this is done.

**Comment by FrelliBB** [comment]
Isn't this what the embedded Sushi API is for?

**Comment by KPull** [comment]
Yes and, in fact, this should go along with the embedded test server. I suggested to replicate the Google Books API, because it returns a _massive_ amount of data compared to the Sushi API. There were problems (related to character encoding, for example) when I called the Google Books API. By copying at least one call to our embedded server, we guarantee that the data won't change across different test runs.
